src/ffi/pystate.rs

```diff
@@ -7,10 +8,29 @@ use std::os::raw::{c_int, c_long};
 pub const MAX_CO_EXTRA_USERS: c_int = 255;
 
 #[repr(C)]
-#[derive(Copy, Clone)]
+#[derive(Debug, Copy, Clone)]
 pub struct PyInterpreterState {
```

Comment:
Hmm, this looks dangerously broken, because it doesn't match at all the full contents of the corresponding struct in the Python headers: https://github.com/python/cpython/blob/af5fb6706219d7949c1db5c9f2b7da53198123f3/Include/internal/pycore_pystate.h#L67

Appreciate that this existed before you wrote this PR, but I don't think editing this struct in this PR achieves much.

As it's an internal type (and probably very different on PyPy),I think we should just replace with an opaque type:

```
opaque_struct!(PyInterpreterState);
```

Reply:
Done.